tests/api_connexion/endpoints/test_dag_run_endpoint.py

```diff
@@ -29,28 +35,555 @@ def setUpClass(cls) -> None:
 
     def setUp(self) -> None:
         self.client = self.app.test_client()  # type:ignore
+        self.now = timezone.utcnow()
```

Comment:
I would prefer to use a fixed date in tests. This allows us to use text strings in tests.  Now it is very difficult for me to check if the data format is correct.